src/build/__main__.py

```diff
@@ -121,25 +145,39 @@ def _handle_build_error() -> Iterator[None]:
             print()
         else:
             if e.exc_info:
-                traceback.print_exception(
+                tb_lines = traceback.format_exception(
                     e.exc_info[0],
                     e.exc_info[1],
                     e.exc_info[2],
                     limit=-1,
                 )
+                tb = ''.join(tb_lines)
             else:
-                print(traceback.format_exc(-1))
+                tb = traceback.format_exc(-1)
+            _print('\n[dim]{}[reset]\n'.format(tb.strip('\n')))
```

Comment:
Here's where it looks like `[<stuff>]` could get introduces in the tb and miss-processed.